internal/bzlmod/go_deps.bzl

```diff
@@ -592,6 +711,10 @@ _module_tag = tag_class(
         ),
         "build_naming_convention": attr.string(doc = """Removed, do not use""", default = ""),
         "build_file_proto_mode": attr.string(doc = """Removed, do not use""", default = ""),
+        "_parent_label": attr.label(
```

Comment:
Instead of putting this on all tags, could you make it a property of the synthetic tags only that we test for with `getattr`? I don't know how Bzlmod handles tag attributes with a `_` prefix - they may still be settable.

Reply:
I believe this is also used for non-synthetic tags. But I need to refresh my memory.... if it is used for non-senthetic, could we keep it?